Resources/Prototypes/_Moffstation/Entities/Clothing/Uniforms/jumpsuits.yml

```
@@ -73,3 +73,16 @@
     - ClothMade
     - PrisonUniform
     - WhitelistChameleon
+
+- type: entity
+  parent: UnsensoredClothingUniformBase # Moffstation - (No Sensors for Enemy Gear)
```

```suggestion
  parent: UnsensoredClothingUniformBase
```
We're already in a moffstation directory